apps/backend/src/api/devices.py

```
@@ -110,17 +112,31 @@ async def delete_device_by_hostname(
             success=True,
+            operation_id=operation_id,
             operation_type="delete_device",
             result=result,
+            error_message=None,
+            warnings=[],
+            execution_time_ms=execution_time_ms,
             message=f"Device '{hostname}' deleted successfully"
         )
     except DeviceNotFoundError as e:
+        execution_time_ms = int((time.time() - start_time) * 1000)
         raise HTTPException(status_code=404, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)